Resources/Private/Content/Sites.xml

```diff
@@ -1142,7 +1142,7 @@
            <text __type="string">&lt;p&gt;
                                                 &lt;/p&gt;&lt;p&gt;Don't explain anymore how editing works. Just
                                                 discover it yourself! All contents are directly edited on the page, just
-                                                styled like the live content. Powered by Aloha Editor and Hallo Editor,
+                                                styled like the live content. Powered by CKEditor,
                                                 you'll directly see what you write, just as the end-user does.&lt;/p&gt;&lt;p&gt;Furthermore,
                                                 the editing is based on semantic technologies such as RDFa, which we
                                                 consume using VIE and CreateJS. This allows to expose the whole content
```

Comment:
is this still correct?

Reply:
Probably not… @markusguenther / @dimaip !?